peepsView.js

```diff
@@ -0,0 +1,143 @@
+class PeepsView {
+  constructor(model, api) {
+    this.user_id = "";
```

Comment:
I saved this data to model rather than view as model is for saving data and view is for displaying it, not sure if that's correct though